app/src/main/res/layout-sw600dp-port/onboarding_slide.xml

```diff
@@ -41,7 +41,7 @@
         android:layout_marginTop="48dp"
         android:layout_marginEnd="160dp"
         android:gravity="center_horizontal"
-        android:maxLines="1"
+        android:maxLines="2"
```

Comment:
Same previous comment applies to this code too.

Remove maxLines attribute.

Also, please re-generate screenshots and update PR description just to be sure

Reply:
Noted.